kubejs/server_scripts/enigmatica/kubejs/base/entity/on_death.js

```diff
@@ -0,0 +1,30 @@
+{
+  const bee_map = {}
```

Comment:
I'd prefer this const inside the `onEvent` call, that's how we usually do with non-global consts

Reply:
but then it would get redefined **each time** any mob dies, better to initialize the map outside the death event  👀

Reply:
Very valid point 👍